configs:
  mode:
    runner: "offvid"
    sequentialSubtraction: false
  sensor:
    general:
      fpsBoost: true
      brightness:
        alpha: 1.0
        beta: 0.0
    offline:
      rosbag:
        raw_image_topic: "/camera/color/image_raw"
  algorithm:
    process:
      alignment:
        matchRate: 0.4
        usePreset: true
        maxFeatures: 500
      channel: "g" # ["b", "g", "r", "all"]
      subtractRL: true
    postprocess:
      threshold:
        size: 50
        method: "binary" # ["binary", "otsu", "both"]
      invertBinary: true
      erosionKernelSize: 1
      gaussianKernelSize: 1
  marker:
    structure:
      size: 0.1
    detection:
      dictionary: "DICT_ARUCO_ORIGINAL"
